ios/FluentUI/List/ListItem.swift

```diff
@@ -0,0 +1,154 @@
+                            .frame(minHeight: ListItemTokenSet.subtitleThreeLineHeight)
+                        Text(footer)
+                            .foregroundColor(Color(uiColor: tokenSet[.footerColor].uiColor))
+                            .font(Font(tokenSet[.footerFont].uiFont))
+                            .frame(minHeight: ListItemTokenSet.footerHeight)
+                            .lineLimit(state.footerLineLimit)
+                            .truncationMode(state.footerTruncationMode)
+                    }
+                }
+            }
+        }
+
+        @ViewBuilder
+        var accessoryView: some View {
+            if let accessoryType = state.accessoryType,
+               let icon = accessoryType.icon,
+               let iconColor = accessoryType.iconColor(tokenSet: tokenSet, fluentTheme: fluentTheme) {
+                let image = Image(uiImage: icon)
+                    .foregroundColor(Color(uiColor: iconColor))
+                if accessoryType == .detailButton {
+                    SwiftUI.Button {
+                        if let onAccessoryTapped = state.onAccessoryTapped {
+                            onAccessoryTapped()
+                        }
+                    } label: {
+                        image
+                    }
+                } else {
+                    image
+                }
+            }
+        }
+
+        @ViewBuilder
+        var backgroundView: some View {
+            if let backgroundColor = state.backgroundStyleType.defaultColor(tokenSet: tokenSet) {
+                Color(backgroundColor)
+            }
+            EmptyView()
+        }
+
+        @ViewBuilder
+        var contentView: some View {
+            HStack(alignment: .center) {
+                leadingView()
+                    .padding(.trailing, ListItemTokenSet.horizontalSpacing)
+                labelsView
+                    .padding(.trailing, ListItemTokenSet.horizontalSpacing)
+                Spacer()
+                trailingView()
+                    .tint(Color(fluentTheme.color(.brandForeground1)))
+                accessoryView
+                    .padding(.leading, ListItemTokenSet.horizontalSpacing)
+            }
+            .padding(EdgeInsets(top: ListItemTokenSet.paddingVertical + 1,
+                                leading: ListItemTokenSet.paddingLeading,
+                                bottom: ListItemTokenSet.paddingVertical + 1,
+                                trailing: ListItemTokenSet.paddingTrailing))
+            .frame(minHeight: minHeight)
+            .background(backgroundView)
+            .listRowInsets(EdgeInsets())
+        }
+        
+        return contentView
+    }
+
+    var state: ListItemState = ListItemState()
+    
+    private var minHeight: CGFloat {
+        if !subtitle.isEmpty {
+            if !footer.isEmpty {
+                return ListItemTokenSet.threeLineMinHeight
```

Comment:
Do we need/want any guards against setting the footer but not the subtitle? It looks like it'd be possible to set the footer and the title, and then the min height would be for one line.

Reply:
So I have a similar check in the labels stack, so a footer cant be shown if a subtitle is empty.

I was thinking about introducing an enum like in MSFTableViewCell for layoutType:
``` .swift
    fileprivate enum LayoutType {
        case oneLine
        case twoLines
        case threeLines
    }
```

So I could have a computed variable to determine the layoutType based title, subtitle, and footer values.